build/stack-local.yaml

```diff
@@ -0,0 +1,52 @@
+ghc-options:
+    $locals:
+        -DFastCompilation
+        -fenable-rewrite-rules
+        -threaded
+        -funbox-strict-fields
+        -O0
+        -fconstraint-solver-iterations=100
+    $everything:
+        -DFastCompilation
+        -fenable-rewrite-rules
+        -threaded
+        -funbox-strict-fields
+        -O0
+        -fconstraint-solver-iterations=100
+local-bin-path: ../dist/bin/public/luna
+packages:
+    - {location: ./shell}
```

Comment:
I think there's no need to put them in curly brackets, as well as using `./dir` instead of `dir`